cmake/FindCryptoMiniSat.cmake

```diff
@@ -13,16 +13,78 @@
 # CryptoMiniSat_INCLUDE_DIR - the CryptoMiniSat include directory
 # CryptoMiniSat_LIBRARIES - Libraries needed to use CryptoMiniSat
 
+include(deps-helper)
 
 find_path(CryptoMiniSat_INCLUDE_DIR NAMES cryptominisat5/cryptominisat.h)
```

Comment:
CryptoMiniSat exports CMake targets when installed. We should use `find_package(cryptominisat5)`with the specific version we want in this case.

Reply:
You are right. I'm working on it (see https://github.com/msoos/cryptominisat/pull/645)